src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -520,9 +538,16 @@ export default class DefaultDeviceController implements DeviceControllerBasedMed
         newDevice.stream = stream;
         newDevice.constraints = proposedConstraints;
       } else {
+        //Check can be removed after we confirm that getUserMedia bug has been fixed by chromium
```

Comment:
Need a link to the actual Chrome bug. And please put it in a format TODO: remove when this bug is fixed.